src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -527,13 +540,14 @@ internal void RaiseEvent(string name, string input)
                 }
                 else
                 {
-                    // The orchestrator was not waiting for any event by this name, so the event will be dropped.
-                    this.config.TraceHelper.ExternalEventDropped(

```

Comment:
Yay! Since we're not dropping events anymore, can we repurpose this trace to say that an event has been buffered?